packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-pure/built-in-static-methods-proposals/output.js

```diff
@@ -2,7 +2,6 @@ var _Array$from = require("core-js-pure/features/array/from.js");
 var _Array$isArray = require("core-js-pure/features/array/is-array.js");
 var _Array$of = require("core-js-pure/features/array/of.js");
 var _Date$now = require("core-js-pure/features/date/now.js");
-var _JSON$stringify = require("core-js-pure/features/json/stringify.js");
```

Comment:
It should not be removed with [those](https://github.com/babel/babel-polyfills/blob/main/packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-pure/built-in-static-methods-proposals/options.json) options.

Reply:
I think it's removed because it seems to default to version `3.0`. I noticed this while testing the package.

Reply:
[This entry is available in `3.0`](https://unpkg.com/core-js-pure@3.0.1/features/json/stringify.js).

IIRC it could be removed because of the order of modules in the array of dependencies - here the first is `es.json.stringify` that wasn't in `3.0` - however, it does not mean that `core-js-pure/features/json/stringify` is not required.

Reply:
It checks `getModulesByVersion` (something like that, which references a JSON file with that metadata) in `core-js-compat` and `es.json.stringify` is not mentioned under `3.0`. I think that's why.

Reply:
Yes, because `JSON.stringify` polyfill was only in `es.symbol` and it was exported in `core-js-pure/features/json/stringify`.